packages/app/package.json

```diff
@@ -7,7 +7,7 @@
     "node": ">=18.12.0"
   },
   "scripts": {
-    "dev": "NEXT_PUBLIC_SERVER_URL=http://localhost:8000 next dev -p 8080",
+    "dev": "next dev",
```

Comment:
don't we want to keep port 8080?

Reply:
port 8080 is specified in the env file, where the only place to configure ports going forward